config.m4

```diff
@@ -100,7 +100,7 @@ if test "$PHP_XDEBUG" != "no"; then
   XDEBUG_DEVELOP_SOURCES="src/develop/develop.c src/develop/monitor.c src/develop/php_functions.c src/develop/stack.c src/develop/superglobals.c"
   XDEBUG_GCSTATS_SOURCES="src/gcstats/gc_stats.c"
   XDEBUG_PROFILER_SOURCES="src/profiler/profiler.c"
-  XDEBUG_TRACING_SOURCES="src/tracing/trace_computerized.c src/tracing/trace_html.c src/tracing/trace_textual.c src/tracing/tracing.c"
+  XDEBUG_TRACING_SOURCES="src/tracing/trace_flamegraph.c src/tracing/trace_computerized.c src/tracing/trace_html.c src/tracing/trace_textual.c src/tracing/tracing.c"
```

Comment:
Can you leave this alphabetised please?

Reply:
Yes of course, I had no idea you did keep it alphabetised.